src/components/App/Post/PostQuickAdd.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { uniqueId } from 'lodash';
+
+import { Input } from '../../Form';
+
+const generatePostUrl = () => {
```

Comment:
This could be in a separate file

Reply:
Yes, we talked about that too, like src/utils/index.js

Reply:
And the name should be changed to "generatePostImageUrl"